libcodechecker/server/server.py

```diff
@@ -380,10 +377,22 @@ def do_POST(self):
             return
 
         except Exception as exn:
-            import traceback
-            traceback.print_exc()
-            LOG.error(str(exn))
-            self.send_error(404, "Request failed.")
+            # Convert every Exception to the proper format which can be parsed
+            # by the thrift clients before sending back as a response to a
+            # request.
+            ex = TApplicationException(TApplicationException.INTERNAL_ERROR,
+                                       exn.message)
+            (fname, mtype, seqid) = iprot.readMessageBegin()
+            oprot.writeMessageBegin(fname, TMessageType.EXCEPTION, seqid)
```

Comment:
Once I monkey together a client which disregards the fact that I try to use a nonexistent product, the server handler throws an exception that `TMessageType` is not imported.